apps/microsoft/src/inngest/functions/users/sync-users.ts

```diff
@@ -18,59 +18,63 @@ const formatElbaUser = (user: MicrosoftUser): User => ({
   additionalEmails: [],
 });
 
-/**
- * DISCLAIMER:
- * This function, `syncUsersPage`, is provided as an illustrative example and is not a working implementation.
- * It is intended to demonstrate a conceptual approach for syncing users in a SaaS integration context.
- * Developers should note that each SaaS integration may require a unique implementation, tailored to its specific requirements and API interactions.
- * This example should not be used as-is in production environments and should not be taken for granted as a one-size-fits-all solution.
- * It's essential to adapt and modify this logic to fit the specific needs and constraints of the SaaS platform you are integrating with.
- */
-export const syncUsersPage = inngest.createFunction(
+export const syncUsers = inngest.createFunction(
   {
-    id: 'microsoft/sync-users-page',
+    id: 'microsoft/sync-users',
```

Comment:
```suggestion
    id: 'microsoft-sync-users',
```

All the function id should follow this pattern `microsoft-foo-bar`